tekton/resources/images/ko-multi-arch-template.yaml

```diff
@@ -77,8 +77,9 @@ spec:
           script: |
             #!/usr/bin/env sh
 
-            gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}
-            gcloud auth configure-docker
+            IMAGE_REGISTRY_USER=tekton-robot
+            IMAGE_REGISTRY_TOKEN=$(cat "$IMAGE_REGISTRY_CREDENTIALS")
+            echo "$IMAGE_REGISTRY_TOKEN" | docker login ghcr.io -u ${IMAGE_REGISTRY_USER} --password-stdin
```

Comment:
There's no need to `cat` the token to a variable and then `echo` it. 
This is potentially problematic because your using `echo` and if the shell was ran with `-x` it would output the token to the log.

Could you please replace those two lines with:

```
docker login ghcr.io -u ${IMAGE_REGISTRY_USER} --password-stdin <"${IMAGE_REGISTRY_CREDENTIALS}"
```